src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -2132,6 +2139,8 @@ public Optional<VarHandleDesc> describeConstable() {
     @Stable
     TypesAndInvokers typesAndInvokers;
 
+    private static final long TYPES_AND_INVOKERS_OFFSET;
+
```

Comment:
Please move this just before the static block that initializes it (like newer versions).